watertap/unit_models/zero_order/fixed_bed_zo.py

```diff
@@ -115,7 +115,7 @@ def ferric_chloride_demand_equation(b, t):
             doc="Pre-exponential factor for activated carbon demand")
         self.activated_carbon_parameter_b = Var(
             units=pyunits.dimensionless,
-            bounds=(0, None),
+            bounds=(None, None),
```

Comment:
Why are you changing this bound? Are there really any cases where this could be negative?

Reply:
Yes all of these parameters are negative in the yamls.